WooCommerce/Classes/ViewRelated/Dashboard/Settings/Plugins/PluginListViewController.swift

```diff
@@ -9,9 +9,9 @@ final class PluginListViewController: UIViewController, GhostableViewController
 
     @IBOutlet private var tableView: UITableView!
 
-    lazy var ghostTableViewController = GhostTableViewController(options: GhostTableViewOptions(cellClass: HeadlineLabelTableViewCell.self,
+    lazy var ghostTableViewController = GhostTableViewController(options: GhostTableViewOptions(displaysSectionHeader: false,
+                                                                                                cellClass: HeadlineLabelTableViewCell.self,
                                                                                                 rowsPerSection: [10],
-                                                                                                tableViewStyle: .grouped,
```

Comment:
❓ Is there a reason for removing this style? I'm not opposed to keeping plain style to make it simple though.

Reply:
Yeah, I removed it because when being `.grouped` it showed some space on top (similar to a header but smaller)